### PR TITLE
Scan exec

### PR DESCRIPTION
**Pull request by benadida**

### CODE REVIEW

**Comment by benadida** [comment]
I'm fixing test failures, they're just the presence of directories in CI that I have to ensure, all tests pass on dev machine when those ballot-storing directories are present.

File: src/scanner.ts [hunk 1]
```diff
@@ -1,33 +1,100 @@
...
+let numBallotsScanned = 0
+
+function setStatus(newStatus: string, newMessage: string) {
+  status = newStatus
+  message = newMessage
+}
+
+export function fileAdded(ballotPath: string) {
+  interpretFile(election, ballotPath, (ballotPath: string, ballot: Ballot) => {
+    // TODO: work on these status messages a bit more
```
Comment (beausmith) [inline src/scanner.ts]: Would love to have issues tracking out TODOS.

File: src/scanner.ts [hunk 2]
```diff
@@ -1,33 +1,100 @@
...
+let message = ''
+let numBallotsScanned = 0
+
+function setStatus(newStatus: string, newMessage: string) {
+  status = newStatus
+  message = newMessage
+}
+
+export function fileAdded(ballotPath: string) {
+  interpretFile(election, ballotPath, (ballotPath: string, ballot: Ballot) => {
```
Comment (beausmith) [inline src/scanner.ts]: I'm concerned about confusion occurring at some point due to the name of the variable changing between `votes` (see: https://github.com/votingworks/module-scan/pull/5/files#diff-a22679c0e1167fd703b79102f0b220adR94) and `ballot` for the second argument of the callback.

Comment (benadida) [reply src/scanner.ts]: good call, I will fix this.

File: src/scanner.ts (hunk 1 above)
Comment (benadida) [reply src/scanner.ts]: 👍

File: index.html [hunk 3]
```diff
@@ -12,7 +12,771 @@
...
     window.setInterval(get_status, 2000)
+
+
+    function doScan() {
+    fetch('/scan/scan', {
+        method: 'POST'
+    })
+    }
+
+    election = {
```
Comment (beausmith) [inline index.html]: Why is this data not in its own `sampleElection.json` file?

File: src/scanner.ts (hunk 1 above)
Comment (beausmith) [reply src/scanner.ts]: `cvrs` 👍

**Pull request merged**